src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +386,44 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the label toggle is now determined based on whether the type is ANONYMOUS or not. This is a good practice as it makes the UI more dynamic and responsive to changes in state.



<!-- This is an auto-generated comment by OSS CodeRabbit -->